auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.kt

```diff
@@ -72,6 +72,25 @@ public class CredentialsManager @VisibleForTesting(otherwise = VisibleForTesting
         scope: String?,
         minTtl: Int,
         callback: Callback<Credentials, CredentialsManagerException>
+    ) {
+        getCredentials(scope, minTtl, emptyMap(), callback)
+    }
+
+    /**
+     * Retrieves the credentials from the storage and refresh them if they have already expired.
+     * It will fail with [CredentialsManagerException] if the saved access_token or id_token is null,
+     * or if the tokens have already expired and the refresh_token is null.
+     *
+     * @param scope    the scope to request for the access token. If null is passed, the previous scope will be kept.
+     * @param minTtl   the minimum time in seconds that the access token should last before expiration.
+     * @param parameters to send with the request as a map of string with the keys as string
```

Comment:
```suggestion
     * @param parameters additional parameters to send in the request to refresh expired credentials
```

Reply:
I've made the changes in 0d28d3164799d560a5a422f7295ceea4ee573aed  because I was not sure if applying the suggestion and then rebasing the branch would maybe break it.